proof/refine/ARM/Schedule_R.thy

```diff
@@ -4104,6 +4104,34 @@ lemma refillUnblockCheck_corres':
       apply (clarsimp simp: obj_at_simps valid_refills'_def ps_clear_domE scBits_simps opt_map_red)
   done
 
+lemma ifCondRefillUnblockCheck_corres:
+  "corres dc
+     (\<lambda>s. case_option True (\<lambda>scp. sc_at scp s \<and> active_sc_valid_refills s
+                                \<and> pspace_aligned s \<and> pspace_distinct s
+      \<and> (((\<lambda>sc. case_option (sc_active sc) \<top> act) |< scs_of2 s) scp)) scp_opt)
```

Comment:
It's going to be tough to fit it all in, but this condition needs better indentation.